pvw/docker/docker-compose-gpu.yaml

```diff
@@ -2,7 +2,7 @@ version: '3.2'
 
 services:
   pvw:
-    image: public.ecr.aws/enlil/paraview_web_repo:${PVW_IMAGE_TAG:-pvw-enlil-egl}
+    image: public.ecr.aws/h3lioviz/paraview_web_repo:${PVW_IMAGE_TAG:-h3lioviz-egl}
```

Comment:
@bmcclellan-cu are we creating a new public ECR with the deploy? Or do we want to stick with what we had? We'll likely need to push/pull some new images if we rename the image repository.

If we keep the old name, then this change can be removed.